packages/react/src/components/CardLink/CardLink.js

```diff
@@ -5,6 +5,7 @@
  * LICENSE file in the root directory of this source tree.
  */
 import { ClickableTile } from 'carbon-components-react';
+import { Image } from 'carbon-components';
```

Comment:
@carthick the import here is incorrect and is causing build issues. Should be coming relatively from `Image` from the react package.